src/Docker.DotNet/Models/Events.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Docker.DotNet.Models
+{
+    public class EventMessage
```

Comment:
Ah. Add a line to specgen.go by /events
`{reflect.TypeOf(events.Message{}), "EventsMessage"},` so that this stays in sync with the docker types. This will also generate for you the Actor struct etc.